clients/web/src/stylesheets/layout/global.styl

```diff
@@ -111,6 +111,20 @@ textarea
 /* Stuff that applies across the top level object lists */
 .g-top-level-list-title-container
   display inline-block
+  overflow hidden
+  white-space nowrap
+  width calc(100% - 350px)
```

Comment:
Yes, this is a CSS3 feature and all modern browsers should support it. If we want to change the view to look different on smaller device sizes, we'd need to use media queries.